dscblog/pages/info/footer.html

```diff
@@ -0,0 +1,237 @@
+{% block title %}
+    <meta charset="UTF-8" />
+    <script src="https://kit.fontawesome.com/a076d05399.js"></script>
+    <script defer src="https://use.fontawesome.com/releases/v5.0.13/js/all.js" integrity="sha384-xymdQtn1n3lH2wcu0qhcdaOpQwyoarkgLVxC/wZ5q7h9gHtxICrpcaSUfygqZGOe" crossorigin="anonymous"></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <body>
+    <footer>
+      <div class="footer-container">
+        <div class="footer-left">
+          <h4>QUICK LINKS</h4>
+          <br/>
+
+          <ul>
+            <li>
+              <a href="https://blog.dsciem.tech/">Home</a>
+            </li>
+
+            <li>
+              <a href="https://blog.dsciem.tech/">Explore</a>
+            </li>
+
+            <li>
+              <a href="https://blog.dsciem.tech/info/terms">Terms and Condition</a>
+            </li>
+
+            <li>
+              <a href="https://blog.dsciem.tech/info/privacy">Privacy Policy</a>
+            </li>
+          </ul>
+        </div>
+
+        <div class="footer-right">
+          <h4>Follow us on</h4>
+          <br/>
+      
+
+          <div class="rounded-social-buttons">
+                    <a class="social-button facebook" href="https://facebook.com/dsciem" target="_blank"><i class="fab fa-facebook-f"></i></a>
+                    <a class="social-button twitter" href="https://twitter.com/dsc_iem" target="_blank"><i class="fab fa-twitter"></i></a>
+                    <a class="social-button linkedin" href="https://www.linkedin.com/company/dsciem" target="_blank"><i class="fab fa-linkedin"></i></a>
+                    <a class="social-button youtube" href="http://www.youtube.com/c/DSCIEM" target="_blank"><i class="fab fa-youtube"></i></a>
+                    <a class="social-button instagram" href="https://instagram.com/dsc_iem" target="_blank"><i class="fab fa-instagram"></i></a>
+                </div>
+          <p>
+              <a href="https://github.com/dsc-iem/blog"><b>Help us make this website better :) </b></a>
+            
+          </p>
+          <p class="footer-text copyright">©DSC IEM Blogs 2020</p>
+        </div>
+      </div>
+    </footer>
+  </body>
+<style>
+  * {
+  margin: 0;
+  padding: 0;
+}
+
+/* Footer */
+body {
+    background: #D0D8DB;
+    padding-top: 60px;
+    padding-bottom: 40px;
+}
+
+textarea {
+    resize: none;
+}
```

Comment:
it's not required for our footer and might interfere with other elements unintentionally